rasa_nlu/data_router.py

```diff
@@ -211,5 +232,10 @@ def start_train_process(self, data, config_values):
         _config["data"] = f.name
         train_config = RasaNLUConfig(cmdline_args=_config)
         logger.info("Training process started")
-        result = do_train(train_config, self.component_builder)
-        return result
+        config = train_config.as_dict()
+        result = self.pool.submit(do_train, config)
```

Comment:
Need also to pickle the `self.component_builder` (mostly a pure dict: shouldn't be too hard)